hardware/products/ucm/intro.md

```diff
@@ -248,7 +283,7 @@ For larger additional components (67.75mm x 47.625mm).
 
 For larger additional components (67.75mm x 63.5mm).
 
-# Related Products
+# Related Products (Accessories)
```

Comment:
Take out the this whole section since we moved them to their own page. Mention that page in the intro at the top though.

Reply:
To make this section smaller, instead of Related Products (Accessories), I will remove the LCD screens and change the header to development tools and only list the the dev board and breakout board and have a link to the accessories page. I think having this in the overview will look odd.